### ISSUE TITLE
Run docker-php-ext-configure causes: Error processing tar file: unexpected EOF

### ISSUE THREAD

**Issue by hieunv-0998** [docker-library/php#974]
**TL;DR:** 
`RUN docker-php-ext-configure` with a newline causes a failed build.
```
ERROR: Service 'php-fpm' failed to build: Error processing tar file(exit status 1): unexpected EOF
```

**Steps to produce:**
1. Create a Dockerfile, which using docker-php-ext-configure with a newline.
 `./project/.docker/php-fpm/Dockerfile`
```Dockerfile
FROM php:7.3-fpm-stretch

RUN apt-get update -yqq && \
    apt-get install -y zlib1g-dev libicu-dev g++

RUN docker-php-ext-configure intl

RUN docker-php-ext-install intl
```
2. Build a new container from above image with docker-compose.
```yaml
version: "3"
services:
  php-fpm:
    container_name: php-fpm
    build:
      context: ./project/.docker/php-fpm/
    volumes:
      - ./project:/var/www/project
    expose:
      - "9000"
```
3. Run `docker-compose` build command.
```
docker-compose up -d --build php-fpm
```
Build fails when execute line: `RUN docker-php-ext-configure intl`
```
Step 2/3 : RUN docker-php-ext-configure intl
 ---> Running in 80f9be8368da
Configuring for:
PHP Api Version:         20180731
Zend Module Api No:      20180731
Zend Extension Api No:   320180731
....
....
creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
ERROR: Service 'php-fpm' failed to build: Error processing tar file(exit status 1): unexpected EOF
```

**How to fix (in my way)**
Append an other command into command `RUN docker-php-ext-configure`
```Dockerfile
RUN docker-php-ext-configure intl && \
    docker-php-ext-install intl
```

I hope this share that helps some person who also is struggling this kind of issue.
But I expect that someone can help me explain why this odd happens, too.

**Comment by hieunv-0998** [comment]
Below are my enviroment information:
```
$ uname -a

Linux b120998-lt 4.15.0-88-generic #88~16.04.1-Ubuntu SMP Wed Feb 12 04:19:15 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

```
$ lsb_release -a

Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
```

```
$ docker version

Client: Docker Engine - Community
 Version:           19.03.7
 API version:       1.40
 Go version:        go1.12.17
 Git commit:        7141c199a2
 Built:             Wed Mar  4 01:22:50 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.7
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.17
  Git commit:       7141c199a2
  Built:            Wed Mar  4 01:21:22 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

```
$ docker info
Client:
 Debug Mode: false

Server:
 Containers: 5
  Running: 0
  Paused: 0
  Stopped: 5
 Images: 6
 Server Version: 19.03.7
 Storage Driver: overlay2
  Backing Filesystem: <unknown>
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.15.0-88-generic
 Operating System: Ubuntu 16.04.6 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 7.653GiB
 Name: b120998-lt
 ID: Z4GP:ZY7E:AL7C:OPCI:S63D:6WH4:SKHE:Z4WY:B4QZ:2R5B:LCBP:PNYQ
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
```

```
$ docker-compose version

docker-compose version 1.24.1, build 4667896b
docker-py version: 3.7.3
CPython version: 3.6.8
OpenSSL version: OpenSSL 1.1.0j  20 Nov 2018
```

**Comment by yosifkit** [comment]
I am unable to reproduce.  That seems to be the end of that layer and so seems more like a docker issue in saving the resulting image layer (and nothing we could do to fix it in the `php` image).  You could try deleting local images and re-pull to see if that clears it up.

<details>
<summary>build output:</summary>

```console
$ docker build -
FROM php:7.3-fpm-stretch

RUN apt-get update -yqq && \
    apt-get install -y zlib1g-dev libicu-dev g++

RUN docker-php-ext-configure intl

RUN docker-php-ext-install intl
Sending build context to Docker daemon  2.048kB
Step 1/4 : FROM php:7.3-fpm-stretch
7.3-fpm-stretch: Pulling from library/php
6d28e14ab8c8: Pull complete 
9ecd958eae23: Pull complete 
4611cd46d612: Pull complete 
ad4a2514121d: Pull complete 
803ced91f371: Pull complete 
d60a85f5b548: Pull complete 
7191fda2ef80: Pull complete 
6f3d2598cec2: Pull complete 
82c549defabe: Pull complete 
6b31c0c04191: Pull complete 
Digest: sha256:f1b2d628ffc5fd53066c6ef83ee6e63924144c166db1e1c24754acde8a033341
Status: Downloaded newer image for php:7.3-fpm-stretch
 ---> 1c4f0ba04ef5
Step 2/4 : RUN apt-get update -yqq &&     apt-get install -y zlib1g-dev libicu-dev g++
 ---> Running in 6ed51d76815d
Reading package lists...
Building dependency tree...
Reading state information...
g++ is already the newest version (4:6.3.0-4).
Suggested packages:
  icu-doc
The following NEW packages will be installed:
  icu-devtools libicu-dev zlib1g-dev
0 upgraded, 3 newly installed, 0 to remove and 1 not upgraded.
Need to get 16.9 MB of archives.
After this operation, 93.6 MB of additional disk space will be used.
Get:1 http://deb.debian.org/debian stretch/main amd64 icu-devtools amd64 57.1-6+deb9u3 [178 kB]
Get:2 http://deb.debian.org/debian stretch/main amd64 libicu-dev amd64 57.1-6+deb9u3 [16.5 MB]
Get:3 http://deb.debian.org/debian stretch/main amd64 zlib1g-dev amd64 1:1.2.8.dfsg-5 [205 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 16.9 MB in 1s (16.4 MB/s)
Selecting previously unselected package icu-devtools.
(Reading database ... 12315 files and directories currently installed.)
Preparing to unpack .../icu-devtools_57.1-6+deb9u3_amd64.deb ...
Unpacking icu-devtools (57.1-6+deb9u3) ...
Selecting previously unselected package libicu-dev.
Preparing to unpack .../libicu-dev_57.1-6+deb9u3_amd64.deb ...
Unpacking libicu-dev (57.1-6+deb9u3) ...
Selecting previously unselected package zlib1g-dev:amd64.
Preparing to unpack .../zlib1g-dev_1%3a1.2.8.dfsg-5_amd64.deb ...
Unpacking zlib1g-dev:amd64 (1:1.2.8.dfsg-5) ...
Setting up icu-devtools (57.1-6+deb9u3) ...
Setting up zlib1g-dev:amd64 (1:1.2.8.dfsg-5) ...
Setting up libicu-dev (57.1-6+deb9u3) ...
Removing intermediate container 6ed51d76815d
 ---> ad6fbb38e551
Step 3/4 : RUN docker-php-ext-configure intl
 ---> Running in 26e802ec6572
Configuring for:
PHP Api Version:         20180731
Zend Module Api No:      20180731
Zend Extension Api No:   320180731
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for a sed that does not truncate output... /bin/sed
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking whether cc understands -c and -o together... yes
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for PHP prefix... /usr/local
checking for PHP includes... -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/lib/php/extensions/no-debug-non-zts-20180731
checking for PHP installed headers prefix... /usr/local/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for re2c... re2c
checking for re2c version... 0.16 (ok)
checking for gawk... no
checking for nawk... nawk
checking if nawk is broken... no
checking whether to enable internationalization support... yes, shared
checking for location of ICU headers and libraries... checking for pkg-config... /usr/bin/pkg-config
found 57.1
found 
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking how to run the C++ preprocessor... g++ -E
checking for ld used by cc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for /usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm -B
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking how to run the C++ preprocessor... g++ -E
checking the maximum length of command line arguments... 1572864
checking command to parse /usr/bin/nm -B output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking if cc supports -fno-rtti -fno-exceptions... no
checking for cc option to produce PIC... -fPIC
checking if cc PIC flag -fPIC works... yes
checking if cc static flag -static works... yes
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
checking for ld used by g++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC
checking if g++ PIC flag -fPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
(cached) (cached) checking how to hardcode library paths into programs... immediate
configure: creating ./config.status
config.status: creating config.h
Removing intermediate container 26e802ec6572
 ---> d8a53b0d5b16
Step 4/4 : RUN docker-php-ext-install intl
...
This step completes as well, but the output is left off for brevity
```

</details>

<details>
<summary>Docker/host info:</summary>

```console
$ docker info
Client:
 Debug Mode: false

Server:
 Containers: 3
  Running: 3
  Paused: 0
  Stopped: 0
 Images: 30
 Server Version: 19.03.5
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: active
  NodeID: kmtwti5wl9lwmdgozgb8nod15
  Is Manager: true
  ClusterID: sejbwq7y8u7abbfs4dhb0zr5a
  Managers: 1
  Nodes: 1
  Default Address Pool: 10.0.0.0/8  
  SubnetSize: 24
  Data Path Port: 4789
  Orchestration:
   Task History Retention Limit: 5
  Raft:
   Snapshot Interval: 10000
   Number of Old Snapshots to Retain: 0
   Heartbeat Tick: 1
   Election Tick: 10
  Dispatcher:
   Heartbeat Period: 5 seconds
  CA Configuration:
   Expiry Duration: 3 months
   Force Rotate: 0
  Autolock Managers: false
  Root Rotation In Progress: false
  Node Address: 192.168.1.7
  Manager Addresses:
   192.168.1.7:2377
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: b34a5c8af56e510852c35414db4c1f4fa6172339
 runc version: 3e425f80a8c931f88e6d94a8c831b9d5aa481657
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 4.9.0-11-amd64
 Operating System: Debian GNU/Linux 9 (stretch)
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 7.75GiB
 Name: minas-morgul
 ID: FKMO:UBPJ:UT3U:BQ4O:ALZK:VBHC:KJ7A:O3YQ:2HHM:OU6Q:7WWD:UFXG
 Docker Root Dir: /mnt/infosiftr/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

$ docker version
Client: Docker Engine - Community
 Version:           19.03.5
 API version:       1.40
 Go version:        go1.12.12
 Git commit:        633a0ea838
 Built:             Wed Nov 13 07:25:58 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.5
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.12
  Git commit:       633a0ea838
  Built:            Wed Nov 13 07:24:29 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
$ uname -a
Linux minas-morgul 4.9.0-11-amd64 #1 SMP Debian 4.9.189-3+deb9u2 (2019-11-11) x86_64 GNU/Linux
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.11 (stretch)
Release:	9.11
Codename:	stretch
```

</details>

**Comment by hieunv-0998** [comment]
@yosifkit 
I deleted local images and re-pull, but it's still the same.
Maybe there's some problem with my OS (Ubuntu 16.04) ?
<details>
  <summary>build output:</summary>

```
$ docker build - 
Step 1/4 : FROM php:7.3-fpm-stretch
 ---> 1c4f0ba04ef5
Step 2/4 : RUN apt-get update -yqq &&     apt-get install -y zlib1g-dev libicu-dev g++
 ---> Running in 9a2d2551edcd
Reading package lists...
Building dependency tree...
Reading state information...
g++ is already the newest version (4:6.3.0-4).
Suggested packages:
  icu-doc
The following NEW packages will be installed:
  icu-devtools libicu-dev zlib1g-dev
0 upgraded, 3 newly installed, 0 to remove and 1 not upgraded.
Need to get 16.9 MB of archives.
After this operation, 93.6 MB of additional disk space will be used.
Get:1 http://deb.debian.org/debian stretch/main amd64 icu-devtools amd64 57.1-6+deb9u3 [178 kB]
Get:2 http://deb.debian.org/debian stretch/main amd64 libicu-dev amd64 57.1-6+deb9u3 [16.5 MB]
Get:3 http://deb.debian.org/debian stretch/main amd64 zlib1g-dev amd64 1:1.2.8.dfsg-5 [205 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 16.9 MB in 8s (1971 kB/s)
Selecting previously unselected package icu-devtools.
(Reading database ... 12315 files and directories currently installed.)
Preparing to unpack .../icu-devtools_57.1-6+deb9u3_amd64.deb ...
Unpacking icu-devtools (57.1-6+deb9u3) ...
Selecting previously unselected package libicu-dev.
Preparing to unpack .../libicu-dev_57.1-6+deb9u3_amd64.deb ...
Unpacking libicu-dev (57.1-6+deb9u3) ...
Selecting previously unselected package zlib1g-dev:amd64.
Preparing to unpack .../zlib1g-dev_1%3a1.2.8.dfsg-5_amd64.deb ...
Unpacking zlib1g-dev:amd64 (1:1.2.8.dfsg-5) ...
Setting up icu-devtools (57.1-6+deb9u3) ...
Setting up zlib1g-dev:amd64 (1:1.2.8.dfsg-5) ...
Setting up libicu-dev (57.1-6+deb9u3) ...
Removing intermediate container 9a2d2551edcd
 ---> 798abd9c5bc7
Step 3/4 : RUN docker-php-ext-configure intl
 ---> Running in b21c23a5b4c0
Configuring for:
PHP Api Version:         20180731
Zend Module Api No:      20180731
Zend Extension Api No:   320180731
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for a sed that does not truncate output... /bin/sed
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking whether cc understands -c and -o together... yes
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for PHP prefix... /usr/local
checking for PHP includes... -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/lib/php/extensions/no-debug-non-zts-20180731
checking for PHP installed headers prefix... /usr/local/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for re2c... re2c
checking for re2c version... 0.16 (ok)
checking for gawk... no
checking for nawk... nawk
checking if nawk is broken... no
checking whether to enable internationalization support... yes, shared
checking for location of ICU headers and libraries... checking for pkg-config... /usr/bin/pkg-config
found 57.1
found 
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking how to run the C++ preprocessor... g++ -E
checking for ld used by cc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for /usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm -B
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking how to run the C++ preprocessor... g++ -E
checking the maximum length of command line arguments... 1572864
checking command to parse /usr/bin/nm -B output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking if cc supports -fno-rtti -fno-exceptions... no
checking for cc option to produce PIC... -fPIC
checking if cc PIC flag -fPIC works... yes
checking if cc static flag -static works... yes
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
checking for ld used by g++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC
checking if g++ PIC flag -fPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
(cached) (cached) checking how to hardcode library paths into programs... immediate
configure: creating ./config.status
config.status: creating config.h
Error processing tar file(exit status 1): unexpected EOF
```
</details>

**Comment by tianon** [comment]
Is your drive full?

**Comment by hieunv-0998** [comment]
@tianon 
No, my drive is not full. 
```
$ df -H
Filesystem      Size  Used Avail Use% Mounted on
udev            4,1G     0  4,1G   0% /dev
tmpfs           822M   19M  803M   3% /run
/dev/sda2       246G   24G  211G  10% /
tmpfs           4,2G  342M  3,8G   9% /dev/shm
tmpfs           5,3M  4,1k  5,3M   1% /run/lock
tmpfs           4,2G     0  4,2G   0% /sys/fs/cgroup
/dev/loop0      190M  190M     0 100% /snap/spotify/36
/dev/loop1       58M   58M     0 100% /snap/core18/1668
/dev/loop2       94M   94M     0 100% /snap/core/8268
/dev/loop3       96M   96M     0 100% /snap/core/8689
/dev/loop4      172M  172M     0 100% /snap/spotify/41
/dev/sda1       536M   38M  499M   8% /boot/efi
tmpfs           822M  3,7M  819M   1% /run/user/319817054
tmpfs           822M   17k  822M   1% /run/user/108
```

**Comment by tianon** [comment]
Sorry for the delay -- unfortunately there's not much more we can do to help (as we can't reproduce).  If you still need help, I'd suggest trying a dedicated support forum such as [the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker).
